app/src/App.js

```diff
@@ -1,53 +1,69 @@
 import React from 'react'
 import { useAragonApi } from '@aragon/api-react'
-import { Main, Button } from '@aragon/ui'
+import {
+  Box,
+  Button,
+  GU,
+  Header,
+  IconMinus,
+  IconPlus,
+  Main,
+  SyncIndicator,
+  Tabs,
+  textStyle,
+} from '@aragon/ui'
 import styled from 'styled-components'
 
 function App() {
-  const { api, appState } = useAragonApi()
+  const { api, appState, path, requestPath } = useAragonApi()
   const { count, isSyncing } = appState
-  console.log(count, isSyncing)
+
+  const pathParts = path.match(/^\/tab\/([0-9]+)/)
+  const pageIndex = pathParts ? parseInt(pathParts[1], 10) - 1 : 0
+
   return (
     <Main>
-      <BaseLayout>
-        {isSyncing && <Syncing />}
-        <Count>Count: {count}</Count>
-        <Buttons>
-          <Button mode="secondary" onClick={() => api.decrement(1).toPromise()}>
-            Decrement
-          </Button>
-          <Button mode="secondary" onClick={() => api.increment(1).toPromise()}>
-            Increment
-          </Button>
-        </Buttons>
-      </BaseLayout>
+      {isSyncing && <SyncIndicator />}
+      <Header
+        primary="Counter"
+        secondary={
+          <>
+            <Button
```

Comment:
Wonder if it's worth leaving these button close by to the counter; I know we don't really do this in apps, but it does make it really obvious they're for incrementing the counter.

We also generally don't put two buttons on the header, so it's arguable if this is better practice ;).

Reply:
Yeah I don’t feel strong either way, I was just trying to make use of the secondary slot of the header :-)

More generally, I was thinking that it could be nice to reshape a bit this app by adding just a bit more things to demo, maybe by doing a todo list instead of a counter? :laughing: 

Taking this example, adding new items would look similar to the way we create votes in the Voting app, and being able to see individual items could be an excuse to demo the paths. And the todo list itself could use DataView maybe :smile:

Reply:
Hmm yes, we could definitely take it further.

I don't know about providing it as a boilerplate though since it'd start introducing more complexity and things to eventually remove.

What if we made more use of tutorial repos, that we also run daily on CI or etc, and added this as a good progression from the simple counter example?